packages/cxl-ui/scss/global/cxl-card.scss

```diff
@@ -0,0 +1,64 @@
+cxl-card {
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  margin: var(--lumo-space-m);
+  overflow: hidden;
+  font-family: inherit;
+  text-overflow: ellipsis;
+  background: var(--lumo-base-color);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius);
+
+  &:hover {
+    border-color: var(--lumo-primary-color);
+  }
+
+  iron-icon {
+    --iron-icon-height: var(--lumo-icon-size-s);
+    --iron-icon-width: var(--iron-icon-height);
+  }
+
+  .entry-header {
+    margin: 0;
+    font-family: inherit;
+    font-style: normal;
+
+    .thumbnail {
+      margin: 0 auto;
+      border: 5px solid var(--lumo-base-color);
+    }
+
+    h4 {
+      padding: 0;
+      margin: 0;
+    }
+
+    p {
+      margin: 0 auto;
+      font-family: inherit;
+      font-size: var(--lumo-font-size-s);
+    }
+  }
+
+  .entry-summary {
+    padding: 0 var(--lumo-space-s);
+
+    p {
+      font-family: inherit;
+    }
+  }
+
+  &[theme~="cxl-testimonial"] {
+    .company-logo {
+      max-width: 100px;
+      margin: var(--lumo-space-s) auto;
+    }
```

Comment:
I think `.company-logo` can be dropped for v1, it's a random invention.

There is one image only, `.thumbnail`.

When more is needed, we'll revisit.